doc/source/user/select-openstack-services.rst

```diff
@@ -1,10 +1,27 @@
-================================
-Select OpenStack services to run
-================================
+=========================
+Select OpenStack Services
+=========================
 
 SUSE Containerized OpenStack currently deploys OpenStack Cinder,
 Glance, Heat, Horizon, Keystone, Neutron, Nova.
 
-You can decide which services to deploy by:
+SUSE Containerized OpenStack deployment will automatically add the following
```

Comment:
this paragraph is about how to access deployed openstack services. maybe belong to the verify/user openstack section in the deployment guide?

Reply:
Added to deploy guide. Left in this location as discussed in the standup. The deployment is an OpenStack installation, the user should know what they're getting.